chiron/tests/test_testsystems.py

```diff
@@ -1,33 +1,130 @@
+    # compare with openmm reference for same system
+    e_ref = compute_openmm_reference_energy(ho, pos)
+    e_ref = e_ref.value_in_unit_system(unit.md_unit_system)
+    assert jnp.isclose(e_chiron, e_ref), "Energy at equilibrium position is not zero"
+
+    pos = jnp.array([[0.0, 0.1, 0.0]])
+    e_chiron = harmonic_potential.compute_energy(pos)
+    # compare with openmm reference for same system
+    e_ref = compute_openmm_reference_energy(ho, pos)
+    e_ref = e_ref.value_in_unit_system(unit.md_unit_system)
+    assert jnp.isclose(e_chiron, e_ref), "Energy at equilibrium position is not zero"
+
+    ###############################
+    # Harmonic OscillatorArray
+    ###############################
+    ho = HarmonicOscillatorArray()
+    # initialize potential
+    from chiron.potential import HarmonicOscillatorPotential
 
-    # don't do anything, just check that we get the right expectation value for the testsystem
-    # NOTE: this doesn't actually test anything meaningful, but it's a good starting point
-    expectation = ho.get_potential_standard_deviation(thermodynamic_state)
-    from openmm import unit
     import jax.numpy as jnp
+    from openmm import unit
 
-    assert jnp.isclose(
-        expectation.value_in_unit_system(unit.md_unit_system), 3.741508178
+    harmonic_potential = HarmonicOscillatorPotential(
+        ho.topology,
+        ho.K,
+        x0=unit.Quantity(
+            jnp.array(
+                [
+                    [0.0, 0.0, 0.0],
+                    [1.0, 0.0, 0.0],
+                    [2.0, 0.0, 0.0],
+                    [3.0, 0.0, 0.0],
+                    [4.0, 0.0, 0.0],
+                ]
+            ),
+            unit.nanometer,
+        ),
     )
+
+    # calculate the energy
+    # at the equilibrium position, the energy should be zero
+    pos = ho.positions.value_in_unit_system(unit.md_unit_system)
+    e_chiron = harmonic_potential.compute_energy(pos)
+
+    assert jnp.isclose(e_chiron, 0.0), "Energy at equilibrium position is not zero"
+    pos = unit.Quantity(
+        jnp.array(
+            [
+                [0.1, 0.0, 0.0],
+                [1.1, 0.0, 0.0],
+                [2.1, 0.0, 0.0],
+                [3.1, 0.0, 0.0],
+                [4.1, 0.0, 0.0],
+            ]
+        ),
+        unit.nanometer,
+    )
+    pos = pos.value_in_unit_system(unit.md_unit_system)
+
+    e_chiron = harmonic_potential.compute_energy(pos)
+
+    e_ref = compute_openmm_reference_energy(ho, pos)
+    e_ref = e_ref.value_in_unit_system(unit.md_unit_system)
+    assert jnp.isclose(e_chiron, e_ref), "Energy at equilibrium position is not zero"
+
+
+def test_LJ_fluid():
+    # initialize testystem
+    from openmmtools.testsystems import LennardJonesFluid
+
+    from chiron.potential import LJPotential
+    from openmm import unit
+
+    lj = LennardJonesFluid()
+    lj_pot = LJPotential()
+    post = lj.positions.value_in_unit_system(unit.md_unit_system)
+    lj_pot.compute_energy(post)
```

Comment:
This test is failing because at the default density of the `LennardJonesFluid` class is 0.05.  So basically all the particles are really spaced out, and not interacting with the default cutoff of 1 nm used in `LJPotential`  In the current logic, this creates two empty arrays for the pairs, which causes the error when trying to evaluate the energy.  I'll modify the pair list generation such that we first check if the list is empty and return 0 if that is the case.

Reply:
also, looks like sigma and epsilon were swapped.